### PR TITLE
Prepare 2.0.3 release

### PR DESCRIPTION
**Pull request by Autophagy**

### CODE REVIEW

File: blackbox/docs/release_notes/2.0.3.txt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+=============
+Version 2.0.3
+=============
+
+Released on 2017/06/30.
+
+.. NOTE::
+
+   If you are upgrading a cluster, you must be running CrateDB
```
Comment (nslater) [inline blackbox/docs/release_notes/2.0.3.txt]: line wrapping looks wrong. please make sure that the whole doc is hard wrapped to 79 chars

File: blackbox/docs/release_notes/2.0.3.txt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+.. WARNING::
+
+   Before upgrading, you should `back up your data`_.
+
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
+
+Changelog
+=========
+
+Breaking Changes
```
Comment (nslater) [inline blackbox/docs/release_notes/2.0.3.txt]: delete unused sections. applies to next section too

File: blackbox/docs/release_notes/2.0.3.txt (hunk 1 above)
Comment (Autophagy) [inline blackbox/docs/release_notes/2.0.3.txt]: This line goes over 79 chars if its not broken here, and I don't think theres a way to split up `:ref:`s over multiple lines :disappointed:

File: blackbox/docs/release_notes/2.0.3.txt [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+Changelog
+=========
+
+Fixes
+-----
+
+ - Fixed support for range queries on the ``_id`` and ``_uid`` columns. These
+   queries were not working since version 2.0.0 (Issue: `#5845`_).
+
+.. _#5845: https://github.com/crate/crate/issues/5845
```
Comment (seut) [inline blackbox/docs/release_notes/2.0.3.txt]: prefer all link definition to be at the bottom of the document

**Pull request merged**